Phonenumber/number_test.go

```diff
@@ -24,7 +24,14 @@ func TestNumber(t *testing.T) {
 			// if we expect an error and there isn't one
 			t.Errorf("FAIL: %s\nNumber(%q): expected an error, but error is nil", test.description, test.input)
 		}
-		//fmt.Println("done")
+	
+	}
+}
+func BenchmarkNumber(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		for _, test := range numberTests {
+			Number(test.input)
+		}
```

Comment:
I am not so sure about this. We need to think about this.

Reply:
ok we will discuss this later

Reply:
all right?